frontend/src/components/StoryForm.tsx

```diff
@@ -62,200 +62,220 @@ const StoryForm: React.FC<StoryFormProps> = ({
-                type="text"
-                value={values.title}
-                onChange={handleChange}
-                onBlur={handleBlur}
-              />
-            </div>
-            <div>
-              {touched.title && errors.title ? (
-                <div className="text-red-500">{errors.title}</div>
-              ) : null}
-            </div>
-            <div className="md:flex items-center justify-between p-1">
-              <label
-                htmlFor="ageGroup"
-                className="font-bold text-lg md:text-xl font-quicksand tracking-normal md:tracking-widest leading-none md:leading-normal lg:leading-loose mr-2"
+              <h1
+                className={`${Heading({
+                  align: "center",
+                })} font-normal text-xl lg:text-3xl text-black font-quicksand tracking-normal md:tracking-widest leading-none md:leading-normal lg:leading-loose`}
               >
-                Age
-                <span className="text-red-500">*</span>
-              </label>
-              <ReusableInput
-                id="ageGroup"
-                name="ageGroup"
-                type="select"
-                value={values.ageGroup}
-                onChange={handleChange}
-                onBlur={handleBlur}
-                options={ageGroupList}
-              />
-            </div>
-            <div>
-              {touched.ageGroup && errors.ageGroup ? (
-                <div className="text-red-500">{errors.ageGroup}</div>
-              ) : null}
+                Create your{" "}
+                <span className="font-pacifico font-bold">Story!</span>
+              </h1>
             </div>
-            <label
-              htmlFor="details"
-              className="font-bold font-quicksand text-pink text-2xl md:text-3xl tracking-normal md:tracking-widest leading-none md:leading-normal lg:leading-loose"
-            >
-              Add more details:
-            </label>
-            <div className="items-center justify-between px-1">
-              <label
-                htmlFor="subject"
-                className="text-lg md:text-xl font-bold font-quicksand"
-              >
-                Subject of the Story
-              </label>
-              <ReusableInput
-                id="subject"
-                name="subject"
-                type="textarea"
-                value={values.subject}
-                onChange={handleChange}
-                onBlur={handleBlur}
-                placeholder="Ex. A young girl named Sara loves exploring the world"
-                rows={2}
-                className="md:w-full"
-              />
-              {touched.subject && errors.subject ? (
-                <div>{errors.subject}</div>
-              ) : null}
+          </div>
+        </div>
+        <div className="grid sm:grid-cols-1 lg:grid-cols-12 gap-x-8">
+          <div className="col-span-7">
+            <div className="bg-kid h-229 background"></div>
```

Comment:
In design, we have a blur background image. I personally like blur backgrounds. If you want to do the same you can add `opacity-25` and see how it looks! Let me know what you think!